src/useChat.ts

```diff
@@ -22,9 +21,9 @@ export const DEFAULT_LAMBDA_VALUE = 0.025
 
 type UseChatConfig = {
   customerId: string;
-  corpusIds: string[];
+  corpusKey: string;
   apiKey: string;
-  enableStreaming?: boolean;
```

Comment:
We should still keep this trigger since this is a functionality choice we want to give the user. We can just keep the else block that uses `sendSearchRequest`.